### DISCUSSION TITLE
part-ii/2017-112/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#32] [Question comments]
Part II 2017 Principles of Quantum Mechanics: Paper 1, Section II, C 

https://questions.tripos.org/part-ii/2017-112/

**Comment by silkca** [comment]
The first part is a totally standard derivation for quantum harmonic oscillators (see e.g. Prof Skinner's notes).

The second part I've totally fucked:

In the first calculation, I find that there is no first-order term, and that the second-order term is something like $\frac{\hbar\omega(n^2-n-1)}{(n!)^2}$ - the only nonzero terms in the second order correction sum are those with m=n and m=n-2.

But for the second calculation, I've just set $m'=\frac{m}{1-2\lambda}$ and $\omega'=\omega\sqrt{1-4\lambda^2}$ which makes $H=\frac{1}{2 m'} \hat{p}^{2}+\frac{1}{2} m' \omega'^{2} \hat{x}^{2}$, $E_n(\lambda)=\hbar\omega\sqrt{1-4\lambda^2}(n+\frac{1}{2})$: this gives no first order term, which is fine, but the second order term is $-4\hbar\omega(n+\frac{1}{2})$, which looks totally different to the first calculation.
